pilot/pkg/security/authz/builder/testdata/action-custom-HTTP-for-TCP-filter-out2.yaml

```diff
@@ -13,5 +13,6 @@ typedConfig:
     envoyGrpc:
       authority: outbound_.9000_._.my-custom-ext-authz.foo.svc.cluster.local
       clusterName: outbound|9000||my-custom-ext-authz.foo.svc.cluster.local
+    timeout: 600s
```

Comment:
wow 600s seems large. What is the default here? Or is it unlimited and we are just trying to make it some finite large number

Reply:
default is 200ms, currently we let the user to use DR to configure the timeout in TCP connection timeout (default 10s) and HTTP idle timeout (default 1hr).